src/index.ts

```diff
@@ -74,6 +75,12 @@ const getOptionalInput = (name: string) => core.getInput(name) || undefined;
       const result = await runPublish({
         script: publishScript,
         githubToken,
+        options: {
+          createGithubReleases:
+            createGithubReleases !== undefined
+              ? Boolean(createGithubReleases)
+              : true,
+        },
```

Comment:
I would refactor this to:
```suggestion
        createGithubReleases: JSON.parse(getOptionalInput('createGithubReleases') ?? 'true'),
```

Any particular reason why you have implemented this as an env variable (`process.env.CREATE_GITHUB_RELEASE`) and not as an input of the action?

Reply:
Not really, I just didn't learn about this way 😓 

Fixed, but I start to wonder where we can document all these action inputs?

Reply:
It should be added and documented~ in this file: https://github.com/changesets/action/blob/f4afbea27e0f68b63e3e018d053582abee1817b6/action.yml

The additional documentation about it in the README would also be cool but we can do that later.

Reply:
Added document in the `action.yml` file